packages/core/package.json

```diff
@@ -55,9 +55,9 @@
       "default": "./components/index.js"
     },
     "./dist/siemens-ix/siemens-ix.css": "./dist/siemens-ix/siemens-ix.css",
-    "./dist/siemens-ix/siemens-ix-core.css": "./dist/siemens-ix/siemens-ix-core.css",
-    "./dist/siemens-ix/theme/classic-dark.css": "./dist/siemens-ix/theme/classic-dark.css",
-    "./dist/siemens-ix/theme/classic-light.css": "./dist/siemens-ix/theme/classic-light.css",
+    "./dist/siemens-ix/siemens-ix-core.css": "./dist-css/siemens-ix-core.css",
+    "./dist/siemens-ix/theme/classic-dark.css": "./dist-css/theme/classic-dark.css",
+    "./dist/siemens-ix/theme/classic-light.css": "./dist-css/theme/classic-light.css",
```

Comment:
`dist-css` is only a temporary build folder. The content will be copied to the dist folder 
https://github.com/siemens/ix/blob/da6d384abc27cdfe4acac1f29e36af49bcf97f4d/packages/core/scripts/build/build-css.ts#L83-L86